tensorflow/buildspec.yml

```diff
@@ -34,21 +34,45 @@
         target: dockerd-entrypoint.py
 
   images:
-    BuildTensorflowCPUTrainPy3DockerImage:
-      <<: *TRAINING_REPOSITORY
-      build: &TENSORFLOW_CPU_TRAINING_PY3 false
-      image_size_baseline: 4899
-      device_type: &DEVICE_TYPE cpu
-      python_version: &DOCKER_PYTHON_VERSION py3
-      tag_python_version: &TAG_PYTHON_VERSION py37
-      tag: !join [ *VERSION, "-", *DEVICE_TYPE, "-", *TAG_PYTHON_VERSION, "-", *OS_VERSION ]
-      docker_file: !join [ docker/, *VERSION, /, *DOCKER_PYTHON_VERSION, /Dockerfile., *DEVICE_TYPE ]
-      context:
-        <<: *TRAINING_CONTEXT
+#    BuildTensorflowCPUTrainPy3DockerImage:
+#      <<: *TRAINING_REPOSITORY
+#      build: &TENSORFLOW_CPU_TRAINING_PY3 false
+#      image_size_baseline: 4899
+#      device_type: &DEVICE_TYPE cpu
+#      python_version: &DOCKER_PYTHON_VERSION py3
+#      tag_python_version: &TAG_PYTHON_VERSION py37
+#      tag: !join [ *VERSION, "-", *DEVICE_TYPE, "-", *TAG_PYTHON_VERSION, "-", *OS_VERSION ]
+#      docker_file: !join [ docker/, *VERSION, /, *DOCKER_PYTHON_VERSION, /Dockerfile., *DEVICE_TYPE ]
+#      context:
+#        <<: *TRAINING_CONTEXT
+#    BuildTensorflowGPUTrainPy3DockerImage:
+#      <<: *TRAINING_REPOSITORY
+#      build: &TENSORFLOW_GPU_TRAINING_PY3 false
+#      image_size_baseline: 4899
+#      device_type: &DEVICE_TYPE gpu
+#      python_version: &DOCKER_PYTHON_VERSION py3
+#      tag_python_version: &TAG_PYTHON_VERSION py37
+#      tag: !join [ *VERSION, "-", *DEVICE_TYPE, "-", *TAG_PYTHON_VERSION, "-", *CUDA_VERSION, "-", *OS_VERSION ]
+#      docker_file: !join [ docker/, *VERSION, /, *DOCKER_PYTHON_VERSION, /, *CUDA_VERSION, /Dockerfile., *DEVICE_TYPE ]
+#      context:
+#        <<: *TRAINING_CONTEXT
+#    BuildTensorflowExampleCu102GPUTrainPy3DockerImage:
+#      <<: *TRAINING_REPOSITORY
+#      build: &TENSORFLOW_GPU_TRAINING_PY3 false
+#      image_size_baseline: 4899
+#      base_image_name: BuildTensorflowGPUCu102TrainPy3DockerImage
+#      device_type: &DEVICE_TYPE gpu
+#      python_version: &DOCKER_PYTHON_VERSION py3
+#      tag_python_version: &TAG_PYTHON_VERSION py37
+#      tag: !join [ *VERSION, "-", *DEVICE_TYPE, "-", *TAG_PYTHON_VERSION, "-", *CUDA_VERSION, "-", *OS_VERSION,
+#                   "-example" ]
+#      docker_file: !join [ docker/, *VERSION, /, *DOCKER_PYTHON_VERSION, /example, /Dockerfile., *DEVICE_TYPE ]
+#      context:
+#        <<: *TRAINING_CONTEXT
```

Comment:
Please delete the entries for images which will not be built by this buildspec.

Please add the CPU image back to the list of images that will be built.

Reply:
As discussed, adding CPU is not needed. Deleted the entries for unused images in buildspec.yml